can_interface: "can0"
# data: [0x01, 0xF6, 0x00, 0x00, 0x00, 0x0A, 0x00, 0x6B]
data: [0x01, 0xFD, 0x01, 0x00, 0x10, 0x01, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x6B]
